src/app/@ansyn/ansyn/modules/overlays/overlay-status/overlay-status.component.ts

```diff
@@ -92,14 +103,6 @@ export class OverlayStatusComponent implements OnInit, OnDestroy, IEntryComponen
 		})
 	);
 
-	@AutoSubscription
```

Comment:
Do not delete. AutoSubscription is great, and this is not the way to deal with it